src/pages/Contributing/Install/index.md

```diff
@@ -30,25 +30,25 @@ please [read the docs](https://github.com/lerna/lerna/) to learn more about it.
 
 ### Clone the repo and start the development environement
 
-clone the repo :  
```

Comment:
don't remove two spaces they are line breaks in markdown

Reply:
Fixed